ml-notebook/binder/environment.yml

```diff
@@ -0,0 +1,86 @@
+# packages in addition to repo2docker/buildpacks/conda/environment.yml
+name: pangeo
+channels:
+  - rapidsai
+  - nvidia
+  - numba
+  - conda-forge
+  - anaconda
+dependencies:
+  - jupyterlab=1
+  - jupyterhub=1
+  - nbgitpuller=0.7
+  - dask=2.4
+  - distributed=2.4
+  - dask-kubernetes=0.9.2
+  - dask-gateway=0.3.0
+  - tornado=6
+  - jupyter-server-proxy=1.1
+  - bokeh=1.3
+  - graphviz=2.40
+  - pip=19
+  # core scipy packages
+  - numpy
+  - scipy
+  - matplotlib
+  - pandas
+  - xarray>=0.13.0
+  - sparse
+  # data science stuff
+  - scikit-image
+  - scikit-learn
+  - dask-ml
+  - tensorflow
+  - tensorflow-gpu
+  - keras
+  - keras-gpu
+  - pytorch-cpu
+  - pytorch-gpu
+  - cupy
+  - cudf
+  - cuml
+  # pyviz
+  - holoviews
+  - panel
+  - geoviews
+  - hvplot
+  - geoviews
+  - datashader
+  # geospatial
+  - iris
+  - cartopy
+  - geopandas
+  - rasterio
+  - netcdf4
+  - h5py
+  - h5netcdf
+  - regionmask
+  - rio-cogeo
+  # geoscience-related
+  - climlab
+  - metpy
+  - satpy
+  - gsw
+  - eofs
+  - esmpy
+  - xesmf
+  - windspharm
+  # intake-related
+  - intake
+  - intake-xarray
+  - intake-esm
+  # zarr-related
+  - zarr
+  - numcodecs
+  - python-blosc
+  - lz4
+  - gcsfs
+  - s3fs
+  - tiledb-py
+  # jupyter-related
+  - ipyleaflet
+  # xarray-related
+  - xgcm
+  - xrft
+  - pip:
+    - git+https://github.com/ian-r-rose/dask-labextension.git@de-asyncify#egg=dask_labextension
```

Comment:
If you're interested in GPU workloads I would also recommend the [nvdashboard](https://pypi.org/project/jupyterlab-nvdashboard/) extension.